### ISSUE TITLE
Installing Virtual Camera Plugin 

### ISSUE THREAD

**Issue by scott802** [stream-labs/slobs-virtual-cam#2]
Hello, I might be missing something very simple. I cannot find a way to install the plug in on my mac once downloaded. Is there a executable file that needs to be run that I am missing?

**Comment by magicmiles** [comment]
yeah im wondering the same thing. how does one install this ? it looks like a command line routine must be run to 'install it'.

**Comment by urmommy-c** [comment]
I will try and find a way too install it

**Comment by urmommy-c** [comment]
git clone https://github.com/stream-labs/slobs-virtual-cam.git
cd slobs-virtual-cam
cd mac
chmod +x install-plugin.sh
thats how far i got

**Comment by jkirsh** [comment]
I'm also having trouble determining how to install it properly.

**Comment by alectrocute** [comment]
+1

**Comment by summeroff** [comment]
This repo isn't meant to be installed on its own. It builds as a submodule of our OBS-Studio fork and installed as part of it. No other installation method is maintained.
